tools/stress/src/org/apache/cassandra/stress/util/JavaDriverClient.java

```diff
@@ -128,13 +132,17 @@ public void connect(ProtocolOptions.Compression compression) throws Exception
             poolingOpts.setMaxRequestsPerConnection(HostDistance.LOCAL, maxPendingPerConnection);
         }
 
-        Cluster.Builder clusterBuilder = Cluster.builder()
-                                                .addContactPoint(host)
-                                                .withPort(port)
-                                                .withPoolingOptions(poolingOpts)
-                                                .withoutJMXReporting()
-                                                .withProtocolVersion(protocolVersion)
-                                                .withoutMetrics(); // The driver uses metrics 3 with conflict with our version
+        Cluster.Builder clusterBuilder = Cluster.builder();
+
+        if (this.cloudConfigFile == null) {
```

Comment:
```suggestion
        if (this.cloudConfigFile == null) 
        {
```

I don't agree with that formatting decision, but to keep the formatting style with the existing code.